nengo_viz/components/raster.py

```diff
@@ -10,6 +10,9 @@
 class Raster(Component):
     def __init__(self, viz, config, uid, obj, n_neurons=None):
         super(Raster, self).__init__(viz, config, uid)
+        if getattr(obj, 'neuron_type') is not None:
```

Comment:
Yeah, we validate so that it can never be None. If it is none, the simulator will never build anyway, so we'll get a ton of issues, not just this one. I say check the probeable list.